docs/components/legend.md

```diff
@@ -0,0 +1,31 @@
+# matestack core component: Legend
+
+Show [specs](/spec/usage/components/legend_spec.rb)
+
+The HTML legend tag implemented in ruby.
+
+## Parameters
+
+This component can take 2 optional configuration params and optional content.
+
+#### # id (optional)
+Expects a string with all ids the main should have.
+
+#### # class (optional)
+Expects a string with all classes the main should have.
+
+## Example
```

Comment:
Please provide another example that can do such as

`legend id: "foo", class: "bar", text: "Hello World"`

or similiar. Then this component can be used as a nice one-liner :)

Reply:
And add a test for it, too